src/game.cpp

```diff
@@ -7184,8 +7184,21 @@ void game::list_items_monsters()
 
 game::vmenu_ret game::list_items( const std::vector<map_item_stack> &item_list )
 {
+    std::vector<map_item_stack> ground_items = item_list;
     int iInfoHeight = std::min( 25, TERMY / 2 );
-    const int width = 44;
+    int width = 44;
+
+    //find max length of item name and resize window width
+    for( const map_item_stack cur_item : ground_items ) {
```

Comment:
```suggestion
    for( const map_item_stack &cur_item : ground_items ) {
```
There is really no need to copy the object.

Reply:
Done. Thanks.